app/deployment/psql.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+pushd ..
+
+# we need the following vars from this .env file:
+# - DATABASE_CONNECTION_STRING
+source backup.prod.env
```

Comment:
could do `source ../backup.prod.env` and skip the pushd/popd.

First I thought the pushd/popd will be confusing when doing local file access from the psql prompt (e.g. `\i`). But then I realize those won't work anyway since those will refer files inside the docker image.

Reply:
It's just for consistency with the other scripts :)